opossum-tool/OpossumUI
# Implement signal search

### Summary of changes

Adds signal search in the locate signals popup. It uses the already existing search function to search for packages that match the search term.

### Context and reason for change

Gives the user another option for locating signals.

### How can the changes be tested

Open the popup with ctrl+L and play around. (For example, try searching for '4.1.1' in the example file - it should give a result where a package has this version.)
